jekyll/_cci2/oss.adoc

```diff
@@ -0,0 +1,120 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Build open source projects
+:description: Best practices for building open source projects
+:experimental:
+:icons: font
+:page-layout: classic-docs
+:page-liquid:
+
+This document provides tips and best practices for building your open source project on CircleCI.
+
+== Credits for open source projects
+
+See the xref:credits#open-source-credit-usage[Using Credits] page for up-to-date information on free credits available for open source projects
+
+[#security]
+== Security
+
+While open source can be a liberating practice, take care not to liberate sensitive information.
+
+* If your repository is public, your CircleCI project and its build logs are also public. Pay attention to the information you choose to print.
+* Environment variables set in the CircleCI application are hidden from the public, these variables will not be shared in <<pass-secrets-to-builds-from-forked-pull-requests,forked PRs>>
+unless explicitly enabled.
+
+[#features-and-settings-for-open-source-projects]
+== Features and settings for open source projects
+
+The following features and settings can be useful for open source projects.
+
+[#private-environment-variables]
+=== Private environment variables
+
+Many projects require API tokens, SSH keys, or passwords. Private environment variables allow you to safely store secrets, even if your project is public.
+
+For more information, see the xref:set-environment-variable#set-an-environment-variable-in-a-project[Set an environment variable] document.
+
+[#only-build-pull-requests]
+=== Only build pull requests
+
+NOTE: The "Only build pull requests" feature is not currently supported for GitHub App projects. To find out if you authorized through the GitHub OAuth app or the CircleCI GitHub App, see the xref:github-apps-integration#[GitHub App integration] page.
```

Comment:
Any reason why not directly use the "standalone-unsupported" snippet here? 
I believe it's not even an option at all in the settings of any standalone project

Reply:
For some reason I thought this option was available to GitLab accounts, that's why I didn't use that snippet. I will check

Reply:
Yep checked, I will update this to use the snippet